vhdbuilder/packer/windows/windows-vhd-configuration.ps1

```diff
@@ -97,7 +97,9 @@ $global:excludeHashComparisionListInAzureChinaCloud = @(
     "azure-vnet-cni-singletenancy-overlay-windows-amd64",
     # We need upstream's help to republish this package. Before that, it does not impact functionality and 1.26 is only in public preview
     # so we can ignore the different hash values.
-    "v1.26.0-1int.zip",
-    "azure-acr-credential-provider-windows-amd64-v1.29.2.tar.gz"
+    "v1.26.0-1int.zip"
+    "azure-acr-credential-provider-windows-amd64-v1.29.12.tar.gz"
```

Comment:
We don't need any credential provider here.
What's required to add here is the artifacts that has a different sha in different cloud because of republishing the artifacts.

Reply:
removed as suggested